src/components/Login/Login.js

```diff
@@ -0,0 +1,51 @@
+//Imports
+import React, { useState } from 'react';
+
+//UI Imports
+import './Login.scss';
+
+const Login = () => {
+  const [validLogin, setValidLogin] = useState(true)
+  const [username, setUsername] = useState('')
+  const [password, setPassword] = useState('')
+
+  //default is true! So no error message appears
+  //if value is blank, then setValidLogin(false)
+  //display error message
+  //include minvalue for security
+
+  const checkLogin = (event) => {
+    if ( username.length > 6 && password.length > 6 ) {
+      event.preventDefault();
+      setValidLogin(false);
```

Comment:
why would the setValidLogin change to false here?

Reply:
This was edited to demonstrate the UI for if a user entered in the incorrect info. I'll need to change it back. Great catch, thank you.